ethereum/contracts/SwapCreator.sol

```diff
@@ -190,26 +235,27 @@ contract SwapCreator is Secp256k1 {
         emit Ready(swapID);
     }
 
-    // Bob can call claim if either of these hold true:
+    // Bob can call `claim` if either of these hold true:
     // (1) Alice has set the swap to `ready` and it's before timeout1
-    // (2) It is between timeout0 and timeout1
+    // (2) It is between timeout1 and timeout2
     function claim(Swap memory _swap, bytes32 _secret) public {
         if (msg.sender != _swap.claimer) revert OnlySwapClaimer();
         _claim(_swap, _secret);
 
-        // send ether to swap claimer
+        // Send ETH to swap claimer if token asset address is zero, otherwise
```

Comment:
```suggestion
        // Send ETH to swap claimer if asset address is zero, otherwise
```

Reply:
I rearranged it a little. The check to see if the address is zero is, arguably, more clear to see from the code than what any comment can provide.